x-pack/filebeat/module/googlecloud/audit/config/pipeline.js

```diff
@@ -123,6 +123,28 @@ function Audit(keep_original_message) {
         }
     };
 
+    // Set event.outcome based on authenticatio_info and status 
+    var setEventOutcome = function(evt) {
+        if (evt.Get("googlecloud.audit.status.code") == null) {
+            var authorization_info = evt.Get("googlecloud.audit.authorization_info");
+            if (authorization_info.length == 1) {
+                if (authorization_info[0].granted == null) {
+                    evt.Put("event.outcome", "unknown");
+                } else if (authorization_info[0].granted == true) {
```

Comment:
I think the comparisons to expected values should use the strict equality check `===`. Like `length === 1`, `granted === true`, and below for status code `=== 0`.